particles/Node.cpp

```diff
@@ -124,10 +125,7 @@ void Node::process(const score::gfx::Message& msg)
     case 2:
         // Name of the mesh
         {
-            //meshName = ossia::convert<std::string>(*val);
-            /*TexturedMeshForParticles newmesh(meshName);
-            mesh->myData = newmesh.myData;
-            mesh->mesh = newmesh.mesh;*/
+            meshName = ossia::convert<std::string>(*val);
```

Comment:
Load our new mesh name. The Node and the Renderer both store the name (this is because they may live in different threads so the Renderer should not just go and try to read these variables at random times, it has to have a copy of them)